apps/web/Dockerfile

```
@@ -4,24 +4,45 @@ FROM nginx:1.27.3-alpine
 WORKDIR /usr/share/nginx/html
 COPY apps/web/dist .
 
-# Create config.js template
-RUN echo 'window.ENV = { \
-  API_URL: "$API_URL", \
-  COLLAB_URL: "$COLLAB_URL", \
-  SUBSCRIPTION_ENABLED: "$SUBSCRIPTION_ENABLED" \
-};' > /usr/share/nginx/html/config.template.js
+# Create config.js template (will be processed by envsubst at container startup)
+RUN cat << 'EOF' > /usr/share/nginx/html/config.template.js
+window.ENV = {
```

Hadolint (2.14.0)

[error] unexpected 'w'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)